templates/terraform/aws/eks/eks_cluster/main.tf

```diff
@@ -345,13 +366,15 @@ resource "kubernetes_secret" "git_secrets" {
 
   for_each = {
     git-addons = {
-      type          = "git"
-      url           = local.gitops_addons_url
+      type = "git"
+      url  = local.gitops_addons_url
+      # comment if you want to uses public repo wigh syntax "https://github.com/xxx" syntax, uncomment when using syntax "git@github.com:xxx"
       sshPrivateKey = data.aws_secretsmanager_secret_version.workload_repo_secret.secret_string
     }
     git-workloads = {
-      type          = "git"
-      url           = local.gitops_workloads_url
+      type = "git"
+      url  = local.gitops_workloads_url
+      # comment if you want to uses public repo wigh syntax "https://github.com/xxx" syntax, uncomment when using syntax "git@github.com:xxx"
```

Comment:
I think we should just always use git@github format for both public and private repos. Not sure if there is any beneficial with or without ssh key